integrations.yml

```diff
@@ -88,3 +88,9 @@
   website: "https://workstack.io"
   icon: "workstack.png"
   category: 3
+-
+  name: "OAuth.io"
+  description: "OAuth that just works ! Integrate easily a Basecamp connect to your web/mobile application in minutes."
```

Comment:
"Ease the OAuth integration of Basecamp API v3 and interact with its API in your web/mobile applications"
Does this sound better ? if yes, I'll commit this version right away :)
If you have any other idea to enhance the description, I'm open to suggestion.